includes/class-wc-payments-features.php

```diff
@@ -393,4 +395,13 @@ private static function is_woopayments_gateway_enabled() {
 
 		return 'yes' === $woopayments_enabled_setting;
 	}
+
+	/**
+	 * Checks if the merchant feedback prompt feature flag is enabled.
```

Comment:
Might as well use this comment to clearly define/document this feature flag.

Is this a temporary feature flag, to allow us to ship the code and opt-in manually to test (e.g. on local or staging sites)? 

When the dust settles on this PR will be useful to document the various flags at plugin and service level, and which are for development purposes vs. to manage eligibility and dismiss state (etc) after launch.

Reply:
I've added a comment here to clarify that this is a temporary, development feature-flag.

Reply:
I've added a comment here and renamed this across the codebase to clarify that this is a temporary, development feature-flag.